week3/homework/src/index.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
+const express = require('express');
+const bodyParser = require('body-parser');
+const {
+  add,
+  readTodo,
+  remove,
+  markAsNotDone,
+  markAsDone,
+  clearTodos,
+  update,
+} = require('./actions');
 
-// TODO: Write the homework code in this file
+const app = express();
+app.use(bodyParser.json());
+
+app.get('/todos/:id', readTodo);
+app.delete('/todos', clearTodos);
+app.post('/todos/add/:text', add);
+app.post('/todos/:id/done', markAsDone);
+app.delete('/todos/:id/done', markAsNotDone);
```

Comment:
Also here, shouldn't be `put` method?

Reply:
Thanks for feedback Samir 👍

Reply:
I made the corrections and push it but it doesn't seem here yet. No idea